src/config/datachart.json

```diff
@@ -1423,14 +1407,6 @@
                 "jp": "$.ig.DataSource が受け入れる有効なデータ ソースまたは $.ig.DataSource 自体のインスタンスが可能です。"
             }
         },
-        "dataSourceType": {
-            "type": "string",
-            "defaultValue": null,
-            "description": {
-                "en": "Explicitly set data source type (such as \"json\"). Please refer to the documentation of $.ig.DataSource and its type property.",
-                "jp": "データ ソースのタイプ (\"json\" など) を明示的に設定します。$.ig.DataSource とそのタイプ プロパティのドキュメントを参照してください。"
-            }
-        },
         "dataSourceUrl": {
```

Comment:
If we remove the response data key and all the related properties which can be configured via the DataSource, we need to remove dataSurceUrl property from the configuration.